core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -214,6 +214,23 @@ ManifestFile writeManifest(Long snapshotId, ManifestEntry<?>... entries) throws
     return writer.toManifestFile();
   }
 
+  @SuppressWarnings("checkstyle:HiddenField")
+  ManifestFile writeDeleteManifest(int formatVersion, Long snapshotId, DeleteFile... deleteFiles)
```

Comment:
Seems like we've already provided a `writeManifest` to write DeleteFiles into delete manifests (https://github.com/apache/iceberg/pull/2271/files#diff-65ca95dcbe2f4bca89bc9454f5434afa4d054a97ae76f01c6a224a41a749a1b2R192) ?   So seems we don't have to provide another `writeDeleteManifest`  ?

Reply:
> Seems like we've already provided a `writeManifest` to write DeleteFiles into delete manifests (https://github.com/apache/iceberg/pull/2271/files#diff-65ca95dcbe2f4bca89bc9454f5434afa4d054a97ae76f01c6a224a41a749a1b2R192) ? So seems we don't have to provide another `writeDeleteManifest` ?

Can't meet the requirements, here needs a Manifest whose content is deleted(https://github.com/apache/iceberg/blob/1dbfe30c4ea9831c87a5f1e704f6b628a04c9c6b/core/src/main/java/org/apache/iceberg/ManifestFiles.java#L145 ).